tests/datastore.py

```diff
@@ -582,6 +582,7 @@
                 "key": "e526243f-f51a-4737-9fea-a5d55f4205fe",
                 "name": "accuracy",
             },
+            "identifier": "performance",
```

Comment:
can you add a test with multiple performances?

Reply:
There is already multiple performances here, as it is the request to get all performances of 1 compute plan. The test would be to add a performance with the same compute task key but this not tested in substra as it is a backend feature.

Reply:
(sorry if I'm being slow, it's Friday afternoon)
but then, if we can cover it at unit test level, why don't we cover it at unit test level? End-to-end tests are all good but they are harder to run, slower and less easy to debug.